.vib/argo-cd/goss/vars.yaml

```diff
@@ -0,0 +1,24 @@
+binaries:
+  - argocd
+  - argocd-server
+  - argocd-util
+directories:
+  - paths:
+    - /.argocd
+    - /app/config/gpg/keys
+    - /opt/bitnami/argo-cd/bin
+    - /opt/bitnami/argo-cd/app
+    - /app/config/gpg/source
+    - /bitnami/argocd
+    mode: "0775"
+  - paths:
+    - /opt/bitnami/argo-cd/app
+    - /opt/bitnami/argo-cd/bin
```

Comment:
Careful, you are checking `/opt/bitnami/argo-cd/bin` and `/opt/bitnami/argo-cd/app` twice.